gtk4/src/param_spec_expression.rs

```diff
@@ -1,26 +1,11 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
+use crate::ParamSpecExpression;
+
 use glib::gobject_ffi;
 use glib::translate::*;
 use glib::ParamSpec;
 
-glib::wrapper! {
-    #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
As those derives are not auto-generated for shared types, i would at least implement Debug manually

Reply:
How do I add those traits?

Reply:
just add the debug one, copy paste the automatically generate Display and replace Display with Debug :)